rollup.config.js

```diff
@@ -111,4 +114,47 @@ export default [
             preserveModules: true,
         },
     },
+    {
+        input: "src/browser.js",
+        plugins: [
+            alias(browserAliases),
+            nodeResolve({
+                browser: true,
+                preferBuiltins: false,
+            }),
+            commonjs({
+                transformMixedEsModules: true,
+            }),
+            json(),
+        ],
+        output: {
+            file: "umd/umd.js",
+            format: "umd",
+            name: "sdk",
+            sourceMap: true,
+        },
+        context: "window",
+    },
+    {
+        input: "src/browser.js",
+        plugins: [
+            alias(browserAliases),
+            nodeResolve({
+                browser: true,
+                preferBuiltins: false,
+            }),
+            commonjs({
+                transformMixedEsModules: true,
+            }),
+            json(),
+            terser(),
+        ],
+        output: {
+            format: "umd",
+            name: "sdk",
+            sourceMap: true,
+            file: "umd/umd.min.js",
```

Comment:
```suggestion
            file: "dist/umd.min.js",
```

Reply:
Fair point, I was looking at https://www.unpkg.com/ to use as CDN. I saw that they recommended using `umd/` but `dist` makes more sense as we might like to use `esm` modules too at some point.

Reply:
Should be resolved by now.